src/server/assets/assetsFilter/index.js

```diff
@@ -0,0 +1,23 @@
+const assetRegex = type =>
+  new RegExp(`(\\/static\\/js\\/(${type})-\\w+\\.\\w+\\.js)`, 'g');
```

Comment:
Sorry, some regex pedantry:
- Several surplus brackets, might be slightly more readable without them
- `g` modifier redundant here unless I'm missing something
- `Regex.prototype.test` may be more efficient if all we care about is whether there is a match (as the engine can skip capturing behaviour)